automation.md

```diff
@@ -0,0 +1,102 @@
+![background](img/logo.png)
+
+[Clicknium](https://www.clicknium.com/) offers automation technology for a variety of uses. It can handle several forms of automation, including SAP automation, Java automation, Windows desktop automation, and image automation. Let's discuss about the automation technology that underlies.
```

Comment:
[Clicknium](https://www.clicknium.com/) provides the automation technology in a variety of scenarios with different    forms, such as ....Let us have a discussion on the automation technology  applied in Clicknium.